torch/testing/_internal/common_nn.py

```diff
@@ -2179,6 +2179,14 @@ def single_batch_reference_fn(input, parameters, module):
         input_fn=lambda: torch.rand(1, 3, 7, 7),
         desc='norm',
     ),
+    dict(
+        module_name='LPPool2d',
+        constructor_args=(1.5, 2),
+        cpp_constructor_args='torch::nn::LPPool2dOptions(1.5, 2)',
+        input_fn=lambda: torch.rand(3, 7, 7),
```

Comment:
nit: why `input_fn` here and `input_size` below?

Reply:
Was trying to stay consistent with the `input_fn` above it. I updated the PR to just use `input_size`.